test/message/test_runner_output.out

```diff
@@ -448,7 +448,51 @@ not ok 55 - custom inspect symbol that throws fail
     }
   code: 'ERR_TEST_FAILURE'
   ...
-not ok 56 - invalid subtest fail
+    not ok 1 - sync throw fails at first
+      ---
+      duration_ms: *
+      failureType: 'testCodeFailure'
+      error: 'thrown from subtest sync throw fails at first'
+      code: ERR_TEST_FAILURE
```

Comment:
Oh, I think the `code` fields here and below need to be in quotes now. The CI is probably going to fail because of it.

Reply:
@cjihrig Oops, I overlooked it. I changed to wrapping error code in single quotes. Thanks again.